integreat_cms/cms/migrations/0034_alter_chatmessage_options.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.15 on 2022-08-26 12:09
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    """
+    Update chat permission field to include default change permission.
+    """
+
+    dependencies = [
+        ("cms", "0033_alter_languagetreenode_visible"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="chatmessage",
+            options={
+                "default_permissions": ("delete", "change"),
+                "ordering": ["-sent_datetime"],
+                "verbose_name": "chat message",
+                "verbose_name_plural": "chat messages",
+            },
+        ),
```

Comment:
```suggestion
        ),
        migrations.RunPython(add_roles, remove_roles),
```
You need to implement to functions to assign/unassign the new permission to users.